### ISSUE TITLE
Consider migrating front-end issues to this repo

### ISSUE THREAD

**Issue by hawkw** [alexandrialibrary/alexandria-frontend#2]
If we end up using this repo for front-end dev, we may want to consider moving all of the front-end tasks from the main Alexandria repo to this repo, so that they can be referenced (and closed) by commits/PRs from this repo

On the other hand, it's nice to have all the issues in one place, so that they can all be part of the same milestones, et cetera. Therefore, we might want to continue tracking all issues on the main Alexandria repo. Alternatively, we could consider tracking tasks/issues externally from GitHub.

**Comment by hawkw** [comment]
Another question: how do we intend on versioning the front-end? Will it be versioned together with the backend or will it be versioned separately? We might want to consider using the same release scheme/milestones for both projects but tracking separate version numbers for the front-end and backend since they are pretty decoupled.

**Comment by redbassett** [comment]
The frontend repo is marked as `experimental` specifically because I'm still figuring these out. Versioning is done most easily in the bower.json file, which already has a version number. I'm good moving all frontend issues to the separate repo. As for milestones, the new frontend code has to go into the backend so it is served once a version is complete, but I'm not sure this process.

**Comment by hawkw** [comment]
Yeah. I know all this stuff is currently experimental, I just meant for eventually. I think the way we'd want to main-line front-end code would be using [git subtree](http://blogs.atlassian.com/2013/05/alternatives-to-git-submodule-git-subtree/) to include the compiled front-end code in the `src/main/webapp` directory in the back-end repo, and updating the subtree commit every time a new front-end version is ready.

**Comment by redbassett** [comment]
We could use a separate repo just for issues/wikis, but I'm fine keeping issues on the appropriate repo until any issues arise (once this repo goes official).

**Comment by hawkw** [comment]
> We could use a separate repo just for issues/wikis 

We could do that. I just like having the issue tracker be on the same repo as the code, so that issues can be referenced/closed from commit messages.

**Comment by hawkw** [comment]
I've added milestones and currently open issues tagged as `frontend` to this repo. Currently, they all just contain pointers back to the associated issues on the main repo.

**Comment by hawkw** [comment]
A lot of the issues are assigned to @redbassett but I'm happy to tackle some of them, as well.
